controllers/mod.js

```diff
@@ -139,23 +126,66 @@ exports.postDeleteMod = (req, res, next) => {
   });
 };
 
-exports.postUpdateMod = (req, res, next) => {
+function removeOldPlants(modId) {
+    IndividualPlant.deleteMany({  module: modId}, (err) => {
+      if (err) { return }
+      return
+    });
+}
+
+  exports.postUpdateMod = (req, res, next) => {
+  let plantLocations = req.body.individualPlants || '{}';
+  ip = JSON.parse(plantLocations)
+  let plant1 = req.body.plant1;
+  let plant2 = req.body.plant2;
+  let plant3 = req.body.plant3;
+  let plant4 = req.body.plant4;
+  let plant5 = req.body.plant5;
 
   Mod.findById(req.body.id, (err, mod) => {
     if (err) { return next(err); }
     mod.x = req.body.x || '';
     mod.y = req.body.y || '';
+    locationCode = req.body.locationCode || '';
     mod.model = req.body.model || '';
     mod.shape = req.body.shape || '';
     mod.notes = req.body.notes || '';
     mod.flipped = req.body.flipped || '';
     mod.tag = req.body.tag || '';
 
     mod.save((err) => {
-      if (err) {
-        if (err) {}
-        return next(err);
-      }
+      if (err) { return next(err);}
+      removeOldPlants(mod.id);
+        for (x in ip) {
+        data = ip[x]
+        if (data.selection == 1) {
+          p = plant1
+        }
+        if (data.selection == 2) {
```

Comment:
fyi each subsequent case should be an `else if` case so that each comparison isn't evaluated, just the first one that matches in the sequence